dotCMS/src/integration-test/java/com/dotcms/MainSuite.java

```diff
@@ -456,7 +458,8 @@
         BundlePublisherTest.class,
         CategoryFactoryTest.class,
         Task210805DropUserProxyTableTest.class,
-        Task210816DeInodeRelationshipTest.class,
+        DotObjectCodecTest.class,
```

Comment:
Is correct the place where these tests are being added? why not at the end? 

Besides, I see two new tests that aren't included here: 
- LettuceCacheTest.java
- RedisPubSubImplTest.java